jenkins/swift-functional-tests/extras.d/55-swift-sproxyd.sh

```diff
@@ -26,10 +26,21 @@ function enable_sproxyd_driver {
     done
 }
 
+function configure_swift_functional_tests {
+    if [[ $DEVSTACK_BRANCH == "stable/kilo" ]]; then
+        # keystone V3 support in devstack working properly
+        testfile=${SWIFT_CONF_DIR}/test.conf
+        iniset ${testfile} func_test auth_version 2
+        iniset ${testfile} func_test auth_prefix /v2.0/
+        iniset /etc/swift/proxy-server.conf DEFAULT disallowed_sections tempurl
```

Comment:
Could you add a comment about why this line has to be added please ? I am not sure myself.